src/dumbo/src/pdu/tcp.rs

```diff
@@ -26,6 +26,8 @@ const FLAGS_AFTER_NS_OFFSET: usize = 13;
 const WINDOW_SIZE_OFFSET: usize = 14;
 const CHECKSUM_OFFSET: usize = 16;
 const URG_POINTER_OFFSET: usize = 18;
+
+// TCP headers end in bytes.
```

Comment:
Options are part of the header so this comment is a bit inaccurate :(

Reply:
This is correct. I removed this comment.